### ISSUE TITLE
Conditional radio buttons not updated

### ISSUE THREAD

**Issue by johnfab** [CalderaWP/Caldera-Forms#3096]
## Do You Need Immediate Help?
Yes

## What Version Of Caldera Forms, WordPress and PHP Are You Using?
WordPress Version: 5.1.1
PHP Version: 7.1.25
MySQL Version: 5.5.5
Caldera Forms Version: 1.8.2

## Does Your Issue Persist When You Disable All Other Plugins and Switch To The Default Theme?
Yes

## What Is The Unexpected Behaviour?
If I have radio buttons that appear through a condition of another radio button, the circle isn't filled when clicked. To say it in other words: the click gets logged but doesn't update the graphic.
Instead it updates when I click another unconditional radio.

Bug can be seen here: https://new-doflores.ch/bestellen/

**Comment by New0** [comment]
Hello @johnfab , thanks for reporting your issue. I could see the issue on your link but can't reproduce it in y environment. Could you share the long debug information ? ( From your WordPress admin at Caldera Forms -> Support under the debug information tab )
And can you share an export of the form ?

**Comment by johnfab** [comment]
Hello @New0 , thanks for the answer. The long report is attached below.

WordPress Version: 5.1.1
PHP Version: 7.1.25
MySQL Version: 5.5.5
Server Software: Apache
Your User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36
Session Save Path: /var/lib/php/session
Session Save Path Exists: No
Session Save Path Writeable: No
Session Max Lifetime: 1440
Opcode Cache:
Apc: No
Memcached: No
Redis: No
Object Cache:
Apc: No
Apcu: No
Memcache: No
Memcached: No
Redis: Yes
WPDB Prefix: lW8S1r_
WP Multisite Mode: No
WP Memory Limit: 40M
Currently Active Theme: Kale: 2.4.2

Plugins:
Caldera Forms: 1.8.3 Active
Classic Editor: 1.4 Not Active
FooGallery: 1.6.17 Not Active
Kirki Toolkit: 3.0.38 Active
NextGEN Gallery: 3.1.11 Active

I tested the whole thing already with all plugins disabled (except Caldera) and the error still exists.
But your answer triggered a new idea: I just switched the Theme from "Kale" back to "Twenty Nineteen" and there everything works as it should!
